src/bundle/Resources/public/js/CKEditor/link/ui/link-form-view.js

```diff
@@ -205,11 +205,13 @@ class IbexaLinkFormView extends View {
         }
 
         const anchorPrefix = '#';
+        const relativeLinkPrefix = '/';
         const schemaPattern = /^[a-z0-9]+:\/?\/?/i;
         const isAnchor = href.indexOf(anchorPrefix) === 0;
+        const isRelativeLink = href.indexOf(relativeLinkPrefix) === 0;
```

Comment:
nitpick: for me, this would be easier to read
```suggestion
        const isRelativeLink = href.startsWith(relativeLinkPrefix);
```